### DISCUSSION TITLE
Make clipboard max_size configurable

### DISCUSSION THREAD

**Discussion by Tiiberiu** [espanso/espanso#1227] [Feature requests and Ideas]
I find myself working with big chucks of text ~2-5k chars. espanso_clipboard handles it well when they are smaller (< 2000), it would be nice to have this limit configurable.

Thank you!

**Comment by federico-terzi** [comment]
Hey @Tiiberiu,

Thank you for the feedback! I'll add it to the roadmap

Cheers :)

**Comment by AucaCoyan** [comment]
We removed the size of the clipboard in #2085
